src/test/scala/com/taxis99/amazon/sqs/SqsClientSpec.scala

```diff
@@ -45,28 +47,28 @@ class SqsClientSpec extends StreamSpec {
     // Produce some message
     aws.sendMessageAsync(qUrl, jsonMsg.toString()).get()
 
-    sqs.consumer(Future.successful(sqsQ), PlayJson) { value =>
+    sqs.consumer(Future.successful(sqsQ)) { value =>
       Future.successful(probe.ref ! value)
     }
 
     probe expectMsg jsonMsg
     succeed
   }
   
-  "#producer" should "produce message to the queue" in withInMemoryQueue { implicit aws =>
+  "#producer" should "create a queue that produces messages to SQS with a promise to indicate when it is done" in withInMemoryQueue { implicit aws =>
```

Comment:
You changed the whole idea of this test, it should be another one instead of a replacement.